### PR TITLE
CNV-37609: Remove unnecessary type selection

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

Since we can get the type of network binding from the NAD, we can remove the type selection as it's not needed.
[UX doc](https://docs.google.com/document/d/1CXuAOGSIgsPCvCkNMyhL4dEvfkZ3OOGPSaids8vgJkI/edit#heading=h.i53x5pbhmdxu)

## 🎥 Demo

Before:
remove-type-dropdown-b4

After:
remove-type-dropdown
remove-type-dropdown2
remove-type-dropdown3

### CODE REVIEW

File: src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx [hunk 1]
```diff
@@ -111,37 +128,72 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
...
+                ideal for L2 devices.
+              </div>
+              <div className="NetworkInterfaceNetworkSelect__popover">
+                <b>SR-IOV binding</b>: Attaches a virtual function network device to the
+                VirtualMachine for high performance.
+              </div>
+              <ExternalLink
+                href="https://docs.openshift.com/container-platform/4.15/virt/vm_networking/virt-networking-overview.html#secondary-network-config"
+                text={t('Learn more')}
+              />
```
Comment (upalatucci) [inline src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx:147]: we can remove the `t` function or move it outside the `Trans`

**Review by hstastna — APPROVED** [review]

File: src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx (hunk 1 above)
Comment (hstastna) [reply src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx:147]: I've LGTMed the PR and then I saw this. Yes, let's remove the `t` function and then we can remove the "hold" label :)

Comment (avivtur) [reply src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx:149]: Moved outside of Trans component. Thanks :)

File: src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx [hunk 2]
```diff
@@ -111,37 +128,74 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
...
-      <FormGroupHelperText validated={validated}>
-        {loaded &&
-          validated === ValidatedOptions.error &&
-          t(
-            'No NetworkAttachmentDefinitions available. Contact your system administrator for additional support.',
-          )}
-      </FormGroupHelperText>
+      {loaded && (
+        <FormGroupHelperText validated={validated}>
+          {validated === ValidatedOptions.error &&
```
Comment (metalice) [inline src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx:193]: move it next to loading? as i will render and empty div

File: src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx [hunk 3]
```diff
@@ -1,19 +1,35 @@
 import React, { Dispatch, FC, MouseEvent, SetStateAction, useEffect, useMemo } from 'react';
+import { Trans } from 'react-i18next';
```
Comment (metalice) [inline src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx:2]: please break this file to small pieces, its getting too big

Comment (avivtur) [reply src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect.tsx]: Moved the popover icon outside

**Pull request merged**